org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/SourceIndexer.java

```diff
@@ -172,7 +173,7 @@ public void resolveDocument() {
 			this.cud.resolve();
 		} catch (Exception e) {
 			if (JobManager.VERBOSE) {
-				e.printStackTrace();
+				Util.log(e);
```

Comment:
Same as above, in this PR either don't touch code functionality inside "debug" or "verbose" blocks, or refactor to log some of the exceptions.